docs/guides/advanced/alternative-clients.md

```diff
@@ -111,4 +115,122 @@ fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 
 ```
 
-Finally, use `cargo run` to execute.
+Set required OTel environment variables, [see above :point_up:](#otel-env-vars).
+
+Finally, run
+
+```bash
+cargo run
+```
+
+to execute.
+
+## Example with Go
+
+First, set up a new Go project:
+
+```sh
+mkdir logfire-go-example && cd logfire-go-example
+go mod init logfire-go-example
+```
+
+Update the `go.mod` to include the following:
+
+```go
+module logfire-go-example
+
+go 1.23.2
+
+require (
+	go.opentelemetry.io/otel v1.30.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.30.0
+	go.opentelemetry.io/otel/sdk v1.30.0
+	go.opentelemetry.io/otel/trace v1.30.0
```

Comment:
1. do these all need to be listed explicitly? aren't some implied by others?
2. do they need to be pinned to specific versions? i don't want this to become outdated with time. if the example code stops working with newer versions i'd like someone to notice and tell us that rather than recommend something that only works with older versions. if we decide we want to pin versions, should we do that in the other languages?